packages/big-design/src/components/Box/spec.tsx

```diff
@@ -7,91 +8,89 @@ import { render } from '@test/utils';
 import { Box } from './index';
 
 test('has margin props', () => {
-  const { container, rerender } = render(<Box />);
+  const { rerender } = render(<Box data-testid="box" />);
```

Comment:
Hmmm... 🤔 we should probably avoid using `data-testid` unless it's absolutely needed.

Maybe we can add some text and query that?
```tsx
  const { rerender } = render(<Box>test</Box>); // screen.getByText('test');
```

Reply:
I think its a good way to use the most appropriate queries. Changed all over the component and updated snapshots